examples/gamma/main.rs

```diff
@@ -114,6 +117,7 @@ pub fn main() {
         encoder.draw(&slice, &pso, &data);
         encoder.synced_flush(&mut graphics_queue, &[&frame_semaphore], &[&draw_semaphore], None);
         swap_chain.present(&mut graphics_queue, &[&draw_semaphore]);
+        graphics_queue.wait_idle();
```

Comment:
I haven't implemented it for the `ll` branch so far (core, not corell), but it's on my todo!